osctiny/extensions/attributes.py

```diff
@@ -0,0 +1,73 @@
+"""
+Attributes extension
+--------------------
+
+.. versionadded:: 0.8.0
+"""
+import typing
+from urllib.parse import urljoin
+
+from lxml.objectify import ObjectifiedElement
+
+from ..utils.base import ExtensionBase
+
+
+class Attribute(ExtensionBase):
+    """
+    Access attribute namespaces and definitions
+    """
+    base_path = "/attribute/"
+
+    def list_namespaces(self) -> typing.List[str]:
+        """
+        Get a list of all namespaces
+
+        :return: List of namespace names
+        """
+        response = self.osc.request(
+            url=urljoin(self.osc.url, self.base_path),
```

Comment:
I don't know for sure, whether the absence of the trailing slash would actually make a difference here, but just in case:
```suggestion
            url=urljoin(self.osc.url, f"{self.base_path}/"),
```

This is not pretty.

If it bothers you enough to reject the whole idea, then that's fine by me.